spectractor/extractor/extractor.py

```diff
@@ -416,9 +430,9 @@ def simulate(self, *params):
         # linear regression for the amplitude parameters
         # prepare the vectors
         self.p = np.asarray(params)
-        A2, D2CCD, dx0, dy0, angle, B, rot, pressure, temperature, airmass, *poly_params = params
-        poly_params_order1 = poly_params[:len(poly_params)//2]
-        poly_params_order2 = poly_params[len(poly_params)//2:]
+        A2, D2CCD, dx0, dy0, angle, B, rot, pressure, temperature, airmass, *poly_params_all = params
+        poly_params = poly_params_all[:len(poly_params_all)//2]
+        poly_params_next_order = poly_params_all[len(poly_params_all)//2:]
```

Comment:
Seeing integer division here scares me a little bit, but I guess if that's guaranteed to always be even by definition then that's fine? (Like if it's some length things that put in there twice, for example). Maybe sort sort of assert/check just after to check nothing went wrong there somehow though?

Reply:
Ok, I add a check with a raise ValueError